src/js/modules/main/main-view-buttons.js

```diff
@@ -0,0 +1,25 @@
+import Button from '../components/button';
+
+function createButton(label, eventHash, eventContext) {
+  const instance = new Button({
+    modifier: 'accept',
+    visible: false,
+    label });
+
+  for (const event in eventHash) {
+    if (!eventHash[event].hasOwnProperty()) {
```

Comment:
I left it because it makes linter happy. Actually at the time of writing it I was thinking about bringing that issue in #133. 

`hasOwnProperty` makes sense in case of more generic iterators to filter out inherited properties.